### ISSUE TITLE
DST Staging Review: Heading levels are hard-coded as h2, h3

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#4237]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** Search Filter
**Staging Review ticket:** [Design System - Search Filter](https://github.com/department-of-veterans-affairs/va.gov-team/issues/109960)

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**This is an issue with the:** code
**High-priority:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

The headings in this component are hard-coded as h2 and h3. This should be a customizable option on the component so that the heading ranks are not out of order when a team uses this component. 

Maintaining a logical heading structure ensures a predictable and consistent experience for users relying on assistive technologies.

## Recommended action
Update the heading levels to be a component property.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG21/#info-and-relationships)
- **Modality:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for DST

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation

**Comment by caw310** [comment]
Closing per PO direction. We will reopen if a team requests this.
